vividus-http-client/src/main/java/org/vividus/http/client/ClientBuilderUtils.java

```diff
@@ -35,24 +34,18 @@ private ClientBuilderUtils()
     {
     }
 
-    public static CredentialsProvider createCredentialsProvider(AuthScope authScope, String usernamePassword)
-    {
-        UserInfo userInfo = UriUtils.parseUserInfo(usernamePassword);
-        return createCredentialsProvider(authScope, userInfo.getUser(), userInfo.getPassword());
-    }
-
-    public static CredentialsProvider createCredentialsProvider(String username, String password)
-    {
-        return createCredentialsProvider(DEFAULT_AUTH_SCOPE, username, password);
-    }
-
     public static CredentialsProvider createCredentialsProvider(AuthScope authScope, String username, String password)
     {
         CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
-        credentialsProvider.setCredentials(authScope, new UsernamePasswordCredentials(username, password));
+        credentialsProvider.setCredentials(authScope, createCredentials(username, password));
         return credentialsProvider;
     }
 
+    public static Credentials createCredentials(String username, String password)
```

Comment:
I guess these methods are used in `HttpClientFactory` only and they are really simple so I propose to inline/move_ them_to_`HttpClientFactory`